_posts/2019-12-05-testowanie-komponentow-z-inputami-i-outputami.md

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
Nadużywasz przecinków ;) Przydałby się jakiś grammar nazi, który by to poprawił.